knative-operator/deploy/resources/knativekafka/broker/eventing-kafka-broker.yaml

```diff
@@ -394,10 +407,16 @@ spec:
               value: "8"
             - name: JAVA_TOOL_OPTIONS
               value: "-XX:+CrashOnOutOfMemoryError"
+
           resources:
             requests:
-              cpu: 200m
-              memory: 300Mi
+              cpu: 1000m
+              # 600Mi for virtual replicas + 100Mi overhead
+              memory: 700Mi
+            limits:
+              cpu: 2000m
+              memory: 1000Mi
```

Comment:
We need to remove the limits, similar to kafka-source-dispatcher

Reply:
Will rerun `make generated-files` when https://github.com/openshift-knative/eventing-kafka-broker/pull/1091 merged